limited/guardsman

```diff
@@ -0,0 +1,24 @@
+**Guardsman** | Townsfolk Killing | Limited
+__Basics__
+Once, the Guardsman may choose to be on guard for the rest of the night. Whilst on guard, they will attack anyone who tries to attack them.
+__Details__
+The Guardsman may choose to be on guard for the rest of the night. This is an immediate ability, and it can only be done once.
+If the Guardsman is attacked while on guard, they survive and attack the attacker. They will be informed that they counterattacked a player, but not whom. The Guardsman can attack multiple times whilst on guard.
```

Comment:
I'm getting suggested to change "whom" to "who" and "whilst" to "while". Can't tell if either of those is a correct or incorrect suggestion

Reply:
those would both be correct, i didn't spot them. will change

Reply:
well i think u meant them the other way around but the other way around is correct
anyway it's currently correct so